cpp/src/io/parquet/page_delta_decode.cu

```diff
@@ -579,15 +579,17 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size)
     __syncthreads();
   }
 
-  // now turn array of lengths into offsets
-  int value_count = nesting_info_base[leaf_level_index].value_count;
+  // now turn array of lengths into offsets, but skip if this is a large string column
```

Comment:
Can you help me understand what is supposed to happen to large strings? There's no "else" clause here.

Reply:
The "else" is handled in `inline_column_buffer::make_string_column_impl()` with the call to `cudf::detail::sizes_to_offsets()`. I could make that clearer in the comment.

Reply:
That helped, thank you!